x11-packages/pcsx-rearmed/build.sh

```diff
@@ -25,6 +25,7 @@ termux_step_configure() {
 
 termux_step_make_install() {
 	install -Dm755 pcsx ${PREFIX}/bin/pcsx
-	mkdir -p ${PREFIX}/etc/pcsx
+	mkdir -p ${PREFIX}/etc/pcsx/plugins
 	cp -r frontend/pandora/skin ${PREFIX}/etc/pcsx/
+	cp plugins/*.so ${PREFIX}/etc/pcsx/plugins/
```

Comment:
Our scripts do not handle ELF objects other than under `bin/`, `lib/` or `libexec/`. For example they are not `strip`ped. Can this be changed? But you can leave it as it is, if it is difficult to change.

Reply:
The plugins directory is parameterized in the [`PLUGINS_DIR` directive](https://github.com/termux/termux-packages/blob/8c6eb2d3cf9c7d0c4385fa61ab0813389657374e/packages/pcsx-rearmed/main.h.patch#L20), so it's easy to change. However, I don't think plugins should belong in any of the given directories.
How about creating a symbolic link to `lib/pcsx_plugins`? Is that fine?

Reply:
If stripping is your only concern, `install -s` is also an option. Do you think it's a viable option?

Reply:
It is OK to install plugins under `$PREFIX/lib/pcsx_plugins` and symlink to that directory from `$PREFIX/etc/[...]`.

`strip`ping in `build.sh` is discouraged because it is likely to be removed by mistake when refactoring. Moreover, in addition to `strip` we use `termux-elf-cleaner` to clean up ELF objects. Once I used it in `build.sh` of a certain package but it was removed afterwards by another maintainer.